blog/llm-finetuning/README.md

```diff
@@ -0,0 +1,34 @@
+# LLM Finetuning using HuggingFace + Determined
+
+First install Determined on your local machine:
+```bash
+pip install determined
+```
+
+Then finetune:
+```bash
+det e create distributed.yaml . 
+```
+
+Change configuration options in `distributed.yaml`.
```

Comment:
I'd add a sentence or two about useful parameters to change (e.g., slots_per_trial).